### ISSUE TITLE
Invalid CSS output when using interpolation with mixins

### ISSUE THREAD

**Issue by Jasonkoolman** [leafo/scssphp#486]
Consider the following SCSS code:

```
@mixin set-border($width) {
    border: $width solid #000;
}

.element {
    @include set-border(2px); // compiles to "2px solid #000"
}
```

Now using interpolation:

```
.element {
    @include set-border(#{2px + 2px}); // compiles to "4pxsolid #000"
}
```

Notice that there is no whitespace between the border width and style declaration.

Hoping this is an easy fix. Thanks in advance 👍

**Comment by a-dg** [comment]
@Jasonkoolman I know this isn't a long-term solution, but changing your mixin to `border: #{$width} solid #000;` solves the issue for me.

**Comment by Cerdic** [comment]
Should be fixed with #694
